src/Attributes/LodataProperty.php

```diff
@@ -41,7 +41,7 @@ public function __construct(
         $alternativeKey = false,
         $searchable = false,
         $filterable = true,
-        ?bool $immutable = false
+        ?bool $immutable = false,
```

Comment:
I love trailing commas myself, but since the library (for some reason) still supports PHP 7.1 this isn't supported...